apollos-church-api/src/data/ActionAlgorithm.js

```diff
@@ -71,6 +72,27 @@ class dataSource extends ActionAlgorithm.dataSource {
     });
   }
 
+  async contentChannelsAlgorithm({ ids }) {
+    const { WCCSeries } = this.context.dataSources;
+    const channels = await Promise.all(
+      ids.map(async (id) => WCCSeries.getFromId(id))
```

Comment:
Wish we could do this in one request :/ I don't see an obvious way to do it w/ their API right now though.